### PR TITLE
Add Path.contains, an API to hit test a Path

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
@abarth 

This is for customer: pink. Pre-req for https://github.com/flutter/flutter/pull/3695.

cc @apwilson

File: sky/engine/core/dart/painting.dart [hunk 1]
```diff
@@ -162,6 +162,13 @@ class Path extends NativeFieldWrapperClass2 {
   /// reset to the origin.
   void reset() native "Path_reset";
 
+  /// Tests to see if the path is within the path. (That is, whether
```
Comment (krisgiesing) [inline sky/engine/core/dart/painting.dart]: I think you mean "the point is within the path"

**Comment by abarth** [comment]
LGTM w/ Kris' comment addressed.

File: sky/engine/core/dart/painting.dart (hunk 1 above)
Comment (Hixie) [inline sky/engine/core/dart/painting.dart]: Wow even after your comment I still read this sentence a half dozen times before realising the problem.

**Pull request merged**